scripts/generate-framework-agreement-signature-pages.py

```diff
@@ -1,16 +1,43 @@
 #!/usr/bin/env python
 """
+Generate framework agreement signature pages from supplier "about you"
+information for suppliers who successfully applied to a framework.
 
-PREREQUISITE: You'll need wkhtmltopdf installed for this to work (http://wkhtmltopdf.org/)
+Usage:
+    scripts/generate-framework-agreement-signature-pages.py [-n] [-v...]
+        <stage> <framework> <output_dir> <agreements_repo>
+        [--supplier-id=<id>... | --supplier-ids-from=<file>]
```

Comment:
The script below is still looking for `<supplier_id_file>`. But changing it below to `<file>` doesn't work for me, neither does changing this line to `<supplier_id_file>`!  Docopt strikes again 😿

Reply:
Good spot, I'll fix that now